_includes/releases/v22.1/v22.1.11.md

```diff
@@ -0,0 +1,59 @@
+## v22.1.11
+
+Release Date: November 14, 2022
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-1-11-security-updates">Security updates</h3>
+
+- HTTP API endpoints under the `/api/v2/` prefix now allow requests through when the cluster is running in insecure mode. When the cluster is running in insecure mode, requests to these endpoints will have the username set to `root`. [#87274][#87274]
+
+<h3 id="v22-1-11-sql-language-changes">SQL language changes</h3>
+
+- Added a new [cluster setting](../v22.1/cluster-settings.html) `cloudstorage.azure.concurrent_upload_buffers` to configure the number of concurrent buffers used when uploading files to Azure. [#90449][#90449]
+
+<h3 id="v22-1-11-db-console-changes">DB Console changes</h3>
+
+- Requests to fetch table and database [statistics](../v22.1/cost-based-optimizer.html#table-statistics) now have limited concurrency. This may make loading the [Databases page](../v22.1/ui-databases-page.html) slower, but in return should result in making those pages less disruptive. [#90575][#90575]
+- Updated the filter labels from **App** to **Application Name** and from **Username** to **User Name** on the [SQL Activity page](../v22.1/ui-overview.html#sql-activity). [#91294][#91294]
+- Fixed the filter and label style on the **Transactions** filter label on the [SQL Activity page](../v22.1/ui-overview.html#sql-activity). [#91319][#91319]
+- Fixed the filters in the DB Console so that if the height of the filter is large, it will allow a scroll to reach **Apply**. [#90479][#90479]
+- Added a horizontal scroll to the table on **Explain Plan** tab under [**Statement Details**](../v22.1/ui-statements-page.html). [#91329][#91329]
+- Fixed the filter height on the [Sessions page](../v22.1/ui-sessions-page.html) so that the full dropdown is viewable with scroll. [#91325][#91325]
+
+<h3 id="v22-1-11-bug-fixes">Bug fixes</h3>
+
+- Fixed an extremely rare out-of-bounds crash in the [protected timestamp](../v22.1/architecture/storage-layer.html#protected-timestamps) subsystem. [#90452][#90452]
+- Fixed the calculation of the `pg_attribute.attnum` column for indexes so that the `attnum` is always based on the order the column appears in the index. Also fixed the `pg_attribute` table so that it includes stored columns in secondary indexes. [#90728][#90728]
+- [TTL job](../v22.1/row-level-ttl.html#view-scheduled-ttl-jobs) decoding error messages now correctly contain hex-encoded key bytes instead of hex-encoded key pretty print output. [#90727][#90727]
```

Comment:
supernit: we tend to hyphenate `pretty-print` in the corpus. But not really many examples either way.